src/Sentry.Extensions.Logging/Sentry.Extensions.Logging.csproj

```
@@ -20,12 +20,15 @@
     <PackageReference Include="Microsoft.Extensions.Http" Version="2.1.0" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
-    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="8.0.0" />
+  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.0'">
```

```suggestion
  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
```

You want this one to be specific to 6.0.0 (since those are the min version u're supporting there).

But on the 8.0 block add the same libraries but with 8.0.0 as a min version.